docs/calicoctl.md

```diff
@@ -0,0 +1,65 @@
+# calicoctl command line interface user guide
+
+The command line tool, `calicoctl`, makes it easy to configure and start Calico
+services and to manage Calico network and security policy.
+
+The tool provides a simple interface for using Calico networking with Docker
+containers.  In addition, many of the calicoctl commands are useful for general
+management of Calico configuration regardless of whether you are running Calico
+on VMs, containers or bare metal.
+
+This user guide is organized in sections based on the top level command options
+of calicoctl.
+
+## Top level help
+
+Run
+
+    calicoctl --help
```

Comment:
I think this might look better if the command is in-line with the text.  Should be able to do that using a back tick.